code/modules/mob/living/simple_animal/hostile/sharks.dm

```diff
@@ -0,0 +1,91 @@
+//shameless copies of carps.
+
+/mob/living/simple_animal/hostile/shark
+	name = "Space Shark"
+	desc = "The best terror of the seas, next to the kraken."
+	icon_state = "shark"
+	icon_living = "shark"
+	icon = 'icons/mob/sharks.dmi'
+	icon_dead = "shark_dead"
+	icon_gib = "carp_gib"
+	environment_smash = 0
+	speak_chance = 0
+	turns_per_move = 3
+	butcher_results = list(/obj/item/reagent_containers/food/snacks/meat = 3)
+	response_help = "pets"
+	response_disarm = "gently pushes aside"
+	response_harm = "hits"
+	speed = 0
+	maxHealth = 75
+	health = 75
+	harm_intent_damage = 18
+	melee_damage_lower = 18
+	melee_damage_upper = 18
+	attacktext = "maims"
+	attack_sound = 'sound/weapons/bite.ogg'
+	gold_core_spawnable = 1
+	//Space shark aren't affected by cold.
+	atmos_requirements = list("min_oxy" = 0, "max_oxy" = 0, "min_tox" = 0, "max_tox" = 0, "min_co2" = 0, "max_co2" = 0, "min_n2" = 0, "max_n2" = 0)
+	minbodytemp = 0
+	maxbodytemp = 1500
+
+	faction = list("shark")
+
+/mob/living/simple_animal/hostile/shark/Process_Spacemove(var/movement_dir = 0)
+	return 1   //No drifting in space for space sharks....either!
+
+/mob/living/simple_animal/hostile/shark/FindTarget()
+	. = ..()
+	if(.)
+		emote("me", 1, "growls at [.]!")
+
+/mob/living/simple_animal/hostile/shark/AttackingTarget()
+	. =..()
+	var/mob/living/carbon/L = .
+	if(istype(L))
+		if(prob(25))
+			L.Knockdown(20)
+			L.visible_message("<span class='danger'>\the [src] knocks down \the [L]!</span>")
+
+
+/mob/living/simple_animal/hostile/shark/laser
+	name = "Laser-Shark"
+	desc = "NOW we've jumped the shark."
+	icon_state = "lasershark"
+	icon_living = "lasershark"
+	icon_dead = "lasershark_dead"
+	icon_gib = "carp_gib"
+	ranged = 1
+	retreat_distance = 3
+	minimum_distance = 0 //Between shots they can and will close in to nash
+	projectiletype = /obj/item/projectile/beam/laser/heavylaser
```

Comment:
Deals 40 harm per shot o_o

Reply:
should be fine since the only places these really spawn is gold slimes/admemes and gold slimes already have stuff like deathbolt/fireball magicarps.